.github/workflows/buildvue.yml

```diff
@@ -1,8 +1,9 @@
 name: Build Vue files
 
 on:
-  pull_request:
-    types: [synchronize, opened]
+  push:
+    paths:
+      - 'plugins/**/vue/src/**'
```

Comment:
This can't work without changing the action, as it currently works with some pull request refs, which aren't available on push